lineman/app/components/group_page/group_page_controller.coffee

```diff
@@ -37,4 +35,15 @@ angular.module('loomioApp').controller 'GroupPageController', ($rootScope, $loca
   @openUploadLogoForm = ->
     ModalService.open LogoPhotoForm, group: => @group
 
+  @handleSubscriptionSuccess = ->
```

Comment:
I imagine that if we had a router we really used then this would be state triggered and would feel less clunky.

I think I'm keen to explore using ui-router